src/types/stagingBuildStatus.ts

```diff
@@ -0,0 +1,13 @@
+export const statusStates = {
+  unSaved: "UNSAVED",
+  pending: "PENDING",
+  ready: "READY",
+  error: "ERROR",
+} as const
+export type buildStatus = typeof statusStates[keyof typeof statusStates]
+
+export interface StagingBuildStatus {
+  commit: string
+  status: buildStatus
+  timeLastSaved: number
```

Comment:
nit: `commit` and `timeLastSaved` doesn't seem to be returned from the backend (was referencing https://github.com/isomerpages/isomercms-backend/pull/1008)

Reply:
Lol ignore, saw that it was fixed in #1636.

Reply:
ya ps shifted my approach halfway